metadata_backend/helpers/parser.py

```diff
@@ -59,7 +117,6 @@ def element_decode(self,
           when there are multiple children with same name - then to list.
         - All "accession" keys are converted to "accesionId", key used by
           this program
-
         Corner cases:
```

Comment:
For consistency, we should add `links` and `studyType` related corner cases here. Or, if we rather want to have them as inline comments, move these corner cases from docstrings to inline comments. I'm fine with both ways, but not having spesific comments both in dosctring and as inline comments.

Reply:
done. added corner cases and removed in-line comments